README.md

```diff
@@ -147,56 +197,46 @@ models:
+    
 ```yml
-# dbt_project.yml
-
-...
 vars:
-  using_address:        false         #disable if you don't have addresses in QuickBooks
-  using_bill:           false         #disable if you don't have bills or bill payments in Quickbooks
-  using_credit_memo:    false         #disable if you don't have credit memos in Quickbooks
-  using_department:     false         #disable if you don't have departments in Quickbooks
-  using_deposit:        false         #disable if you don't have deposits in Quickbooks
-  using_estimate:       false         #disable if you don't have estimates in Quickbooks
-  using_invoice:        false         #disable if you don't have invoices in Quickbooks
-  using_invoice_bundle: false         #disable if you don't have invoice bundles in Quickbooks
-  using_journal_entry:  false         #disable if you don't have journal entries in Quickbooks
-  using_payment:        false         #disable if you don't have payments in Quickbooks
-  using_refund_receipt: false         #disable if you don't have refund receipts in Quickbooks
-  using_transfer:       false         #disable if you don't have transfers in Quickbooks
-  using_vendor_credit:  false         #disable if you don't have vendor credits in Quickbooks
-  using_sales_receipt:  false         #disable if you don't have sales receipts in QuickBooks
-  using_purchase_order: true          #enable if you want to include purchase orders in your staging models
-  using_credit_card_payment_txn: true #enable if you want to include credit card payment transactions in your staging models
+    quickbooks_<default_source_table_name>_identifier: your_table_name 
 ```
 
-## Analysis
+## Analysis of the financial models
 
 After running the models within this package, you may want to compare the baseline financial statement totals from the data provided against what you expect. You can make use of the [analysis functionality of dbt](https://docs.getdbt.com/docs/building-a-dbt-project/analyses/) and run pre-written SQL to test these values. The SQL files within the [analysis](https://github.com/fivetran/dbt_quickbooks/blob/master/analysis) folder contain SQL queries you may compile to generate balance sheet and income statement values. You can then tie these generated values to your expected ones and confirm the values provided in this package are accurate.
 
+# 🔍 Does this package have dependencies?
+This dbt package is dependent on the following dbt packages. Please be aware that these dependencies are installed by default within this package. For more information on the following packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
+> IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them from your root `packages.yml` to avoid package version conflicts.
+
+```yml
+packages:
+    - package: fivetran/quickbooks_source
+      version: [">=0.6.0", "<0.7.0"]
```

Comment:
Will want to bump this based on the source package also being a breaking change

Reply:
updated